stixpy/timeseries/quicklook.py

```diff
@@ -664,9 +601,9 @@ def _parse_hdus(cls, hdulist):
         data = Table(hdulist[2].data)
 
         try:
-            data["time"] = Time(header["date_obs"]) + TimeDelta(data["time"] * u.s)
+            data["time"] = Time(header["date_obs"]) + TimeDelta(data["time"] * u.cs)
         except KeyError:
-            data["time"] = Time(header["date-obs"]) + TimeDelta(data["time"] * u.s)
+            data["time"] = Time(header["date-obs"]) + TimeDelta(data["time"] * u.cs)
```

Comment:
does `data["time"]` not already have a unit? i.e. should it not be 

```suggestion
            data["time"] = Time(header["date-obs"]) + TimeDelta(data["time"])
```